tez-dag/src/main/java/org/apache/tez/dag/app/DAGAppMaster.java

```diff
@@ -2207,7 +2210,9 @@ public Void run() throws Exception {
     }
 
     // Check if the thread dump service is up in any case, if yes attempt a shutdown
```

Comment:
remove thread dump helper related comment, and change to a more generic one that tells we're about to stop hooks if they are running in any case

Reply:
Done
https://github.com/apache/tez/pull/324/commits/943012ab3aeefcfe2e154a669e45c0999c366cb1